Resources/Prototypes/Voice/disease_emotes.yml

```
@@ -52,4 +52,4 @@
 - type: emote
   id: Snore
   category: Vocal
-  chatMessages: [ snores ]
+  chatMessages: [ chat-emote-msg-snore ]
```

yamllint

[error] too many spaces inside brackets

(brackets)